### PR TITLE
Add nb image

### PR DESCRIPTION
**Pull request by dimakis**
Addition of the CodeFlare Operator image stream 

closes https://github.com/opendatahub-io/codeflare-operator/pull/1

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by astefanutti — COMMENTED** [review]
Could the rebase commits and the actual patch be separated so future rebases will be easier.

**Comment by dimakis** [comment]
@astefanutti please sir, more review?

File: config/manifests/bases/params.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+varReference:
```
Comment (astefanutti) [inline config/manifests/bases/params.yaml:1]: I wonder if that's really necessary. But we can move on and check that later.

File: config/manifests/kustomization.yaml [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+    fieldpath: data.namespace
 # These resources constitute the fully configured set of manifests
 # used to generate the 'manifests/' directory in a bundle.
 resources:
-- bases/codeflare-operator.clusterserviceversion.yaml
 - ../default
 - ../scorecard
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+- ../rbac
```
Comment (astefanutti) [inline config/manifests/kustomization.yaml:24]: Why the change from `default` to `rbac`?

**Comment by astefanutti** [comment]
@dimakis just gave it another review cycle.

File: config/manifests/kustomization.yaml (hunk 2 above)
Comment (dimakis) [reply config/manifests/kustomization.yaml:24]: ah but I see, in the kustomize file it references the `../rbac` in there. so it is not needed?

File: config/manifests/bases/params.yaml (hunk 1 above)
Comment (dimakis) [reply config/manifests/bases/params.yaml:1]: TBH I'm not 💯 what this does? and yea I would question if it is necessary
but i'm cool with checking it at a later date 👍

**Review by astefanutti — APPROVED** [review]

**Comment by astefanutti** [comment]
Oh one last thing, can this be squashed to make rebase easier?

File: config/manifests/kustomization.yaml [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
+    fieldpath: data.namespace
 # These resources constitute the fully configured set of manifests
 # used to generate the 'manifests/' directory in a bundle.
 resources:
-- bases/codeflare-operator.clusterserviceversion.yaml
 - ../default
 - ../scorecard
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+- codeflare-notebook-imagestream.yaml
```
[the remark this answers is not in the document]
Comment (astefanutti) [reply config/manifests/kustomization.yaml:24]: @VanillaSpoon that's a good point. @dimakis what's Kustomize behaviour? Does it work that way too?
